Sources/searchable-collection-vc/SearchableCollectionViewController.swift

```diff
@@ -0,0 +1,52 @@
+//
+//  SearchableCollectionViewController.swift
+//  Demo
```

Comment:
Same here, we tend to remove the attribution headers since they don't provide any relevant information.